code/src/cljs/sixsq/nuvla/ui/edge_detail/views.cljs

```diff
@@ -486,8 +400,8 @@
                                        (for [action action-list]
                                          [ui/ListItem {:as  "a"
                                                        :key (str "action." (apply str
-                                                                                  (take 12
-                                                                                        (repeatedly #(char (+ (rand 26) 65))))))}
+                                                                             (take 12
+                                                                               (repeatedly #(char (+ (rand 26) 65))))))}
```

Comment:
Use (random-uuid) instead.
When I click on the menu I go directly to the detail page. I'm not able to click edit button for example

Reply:
this is not part of this PR. I've removed it cause it doesn't make sense to be on the card within the collection view. In the detailed view this also is already covered elsewhere